nio/api.py

```diff
@@ -1066,7 +1066,7 @@ def devices(access_token):
 
     @staticmethod
     def update_device(access_token, device_id, content):
-        # type: (str, Dict[str, str]) -> Tuple[str, str, str]
+        # type: (str, str, Dict[str, str]) -> Tuple[str, str, str]
```

Comment:
Not sure what the preferred standard is atm, but if the codebase is going to Python 3.6+ then you can probably update this to the standard syntax for type hints instead of the mypy Python 2 syntax. 

e.g.
```python
def update_device(access_token: str, device_id: str, content: Dict[str, str]) -> Tuple[str, str, str]:
```

Reply:
@poljar what´s you´re preferred standard for this?

Reply:
Since we dropped support for Python 2 there has been a slowly progressing but ongoing effort to convert this to standard Python 3 syntax. So please no new type hints inside of comments.

Reply:
Done